CMakeLists.txt

```diff
@@ -1,11 +1,16 @@
 cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
 
 cmake_policy(SET CMP0091 NEW)
+if(POLICY CMP0135)
+  cmake_policy(SET CMP0135 NEW)
+endif()
+
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.15" CACHE STRING "Minimum macOS / OS X deployment version")
 
 project(NovelRT
-  VERSION 0.1.0
+  VERSION 1.0.0 #MVP Version
   DESCRIPTION "NovelRT game engine"
-  HOMEPAGE_URL "https://github.com/novelrt/NovelRT"
+  HOMEPAGE_URL "https://novelrt.dev"
```

Comment:
Does this direct to an actual website?

Reply:
This redirect's to the GitHub Organization page we have currently